schema/dereferenced-release-schema.json

```
@@ -9120,6 +9135,22 @@
               "description": "The single amendment object has been deprecated in favour of including amendments in an amendments (plural) array.",
               "deprecatedVersion": "1.1"
             }
+          },
+          "finalStatus": {
+            "title": "Award final status",
+            "description": "The final status of the award, from the closed [awardFinalStatus](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#award-final-status) codelist.",
```

```suggestion
            "description": "The final status of the award, using the closed [awardFinalStatus](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#award-final-status) codelist.",
```
as per https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/schema_style_guide.html#codelists